src/components/atoms/Product/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { ProductType } from "../../../Types";
+import Card from "@material-ui/core/Card";
+import Typography from "@material-ui/core/Typography";
+import { makeStyles } from "@material-ui/core/styles";
+import Button from "@material-ui/core/Button";
+import AddShoppingCartIcon from "@material-ui/icons/AddShoppingCart";
+
+const useStyles = makeStyles({
+  cardContainer: {
+    display: "flex",
+    flexDirection: "column",
+    maxWidth: "300px",
+    margin: "0.5em",
+    padding: "1rem",
+  },
+  textMargin: {
+    margin: "0.25em 0",
+  },
+});
+
+interface Props {
```

Comment:
change the names of the interfaces some think like `I${component name}` ex: IProduct and export the interface